website/client/src/components/admin-panel/index.vue

```diff
@@ -0,0 +1,754 @@
+<template>
+  <div
+    v-if="user.contributor.admin"
+    class="row standard-page"
+  >
+    <div class="well">
+      <h1>Admin Panel</h1>
+      <div class="row">
+        <div class="form-inline col-12">
+          <div class="form-group">
+            <input
```

Comment:
instead of adding two event listeners, one for the input and one for the button we should use a form where the button has `type="submit"` and the user is loaded on submit event

Reply:
I was definitely happy to do that! :) I thought I needed to use similar ways of submitting that were already in use but I'd much rather use normal HTML forms. :) 

Done in 26fdcbbee51937d39a149a20bf7ccd1cf674ca18 and 4a46661fbf6f22d29583fb700634b35d05d9431f but let me know if you want other improvements there.